pkg/modules/npm/handler.go

```diff
@@ -176,16 +176,16 @@ func (m *npm) ListModulesWithDeps(path string) ([]models.Module, error) {
 	if err != nil {
 		return []models.Module{}, err
 	}
-
+	lockFileVersion := pkResults["lockfileVersion"].(float64)
```

Comment:
Why is this float64, and not int32 (or even int8)?
https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json#lockfileversion

Reply:
@RodneyRichardson updated

Reply:
Interesting - I guess that's to cover all situations...